[2024-05-21T03:00:00.880+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-21T03:00:00.897+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_ETL.primera_tarea scheduled__2024-05-21T02:00:00+00:00 [queued]>
[2024-05-21T03:00:00.904+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_ETL.primera_tarea scheduled__2024-05-21T02:00:00+00:00 [queued]>
[2024-05-21T03:00:00.904+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-21T03:00:00.915+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): primera_tarea> on 2024-05-21 02:00:00+00:00
[2024-05-21T03:00:00.919+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=258) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-21T03:00:00.920+0000] {standard_task_runner.py:63} INFO - Started process 259 to run task
[2024-05-21T03:00:00.920+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_ETL', 'primera_tarea', 'scheduled__2024-05-21T02:00:00+00:00', '--job-id', '103', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp2l8i98yo']
[2024-05-21T03:00:00.921+0000] {standard_task_runner.py:91} INFO - Job 103: Subtask primera_tarea
[2024-05-21T03:00:00.950+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_ETL.primera_tarea scheduled__2024-05-21T02:00:00+00:00 [running]> on host 22cda28e7059
[2024-05-21T03:00:01.015+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_ETL' AIRFLOW_CTX_TASK_ID='primera_tarea' AIRFLOW_CTX_EXECUTION_DATE='2024-05-21T02:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-21T02:00:00+00:00'
[2024-05-21T03:00:01.016+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-21T03:00:01.025+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-21T03:00:01.025+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo Iniciando...']
[2024-05-21T03:00:01.033+0000] {subprocess.py:86} INFO - Output:
[2024-05-21T03:00:01.034+0000] {subprocess.py:93} INFO - Iniciando...
[2024-05-21T03:00:01.034+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-21T03:00:01.035+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-21T03:00:01.052+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_ETL, task_id=primera_tarea, run_id=scheduled__2024-05-21T02:00:00+00:00, execution_date=20240521T020000, start_date=20240521T030000, end_date=20240521T030001
[2024-05-21T03:00:01.093+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-21T03:00:01.112+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-21T03:00:01.113+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
